README.md

```diff
@@ -44,14 +44,16 @@ Or use the power of Vue.js within a google map like this:
 <template>
   <gmap-map
     :center="center"
+    @center_changed="center=$event"
```

Comment:
Maybe this is a good distinction about google events change. props changes use camelCase and gogole changes use "_":
* Props change: center-change
* google change event: center_change